modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-secondary-dns-server_{context}"]
+= Configuring DNS server for secondary networks
+
+The Cluster Network Addons Operator (CNAO) deploys the Domain Name Server (DNS) server and monitoring components when you enable the `KubeSecondaryDNS` feature gate in the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to an {product-title} cluster with `cluster-admin` permissions.
+
+.Procedure
+. Create a `LoadBalancer` service using MetalLB or any other load balancer to expose the DNS server outside the cluster. The service listens on port 53 and targets port 5353. For example:
++
+[source,terminal]
+----
+$ oc expose -n openshift-cnv deployment/secondary-dns --name=dns-lb --type=LoadBalancer --port=53 --target-port=5353 --protocol='UDP'
+----
+
+. Retrieve the public IP address of the service by querying the `Service` object:
++
+[source,terminal]
+----
+$ oc get service -n openshift-cnv
+----
++
+.Example output
+[source,terminal]
+----
+NAME        TYPE            CLUSTER-IP     EXTERNAL-IP      PORT(S)          AGE
+dns-lb   LoadBalancer       172.30.27.5    10.46.41.94      53:31829/TCP     5s
+----
+
+. Deploy the DNS server and monitoring components by editing the `HyperConverged` CR:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+    featureGates:
+      deployKubeSecondaryDNS: true <1>
+    kubeSecondaryDNSNameServerIP: "10.46.41.94" <2>
+#...
+----
+<1> Set the `KubeSecondaryDNS` feature gate to `true`.
+<2> Set the IP address of the service to the value retrieved in step 2.
+
+. Retrieve the FQDN of the {product-title} cluster by using the following command:
++
+[source,terminal]
+----
+$ oc get dnses.config.openshift.io cluster -o json | jq .spec.baseDomain
+----
++
+.Example output
+[source,terminal]
+----
+openshift.example.com
+----
+
+. Point to the DNS server using one of the following methods:
+** Add the `kubeSecondaryDNSNameServerIP` value to the `resolv.conf` file on your local machine.
++
```

Comment:
Do we need an example of how to set this parameter?

Reply:
yes, exactly.